trace_decoder/benches/block_processing.rs

```diff
@@ -0,0 +1,49 @@
+//! Benchmarks the processing by the decoder of a block witness obtained from a
+//! node into a sequence of prover inputs ready to be sent to a prover.
+//!
+//! The block being processed here is the 19240650th Ethereum block
+//! (<https://etherscan.io/block/19240650>) containing 201 transactions and 16 withdrawals.
+
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use serde::{Deserialize, Serialize};
+use trace_decoder::{
+    processed_block_trace::ProcessingMeta,
+    trace_protocol::BlockTrace,
+    types::{CodeHash, OtherBlockData},
+};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ProverInput {
+    pub block_trace: BlockTrace,
+    pub other_data: OtherBlockData,
+}
+
+fn resolve_code_hash_fn(_: &CodeHash) -> Vec<u8> {
+    todo!()
+}
+
+fn criterion_benchmark(c: &mut Criterion) {
+    let bytes = std::fs::read("benches/block_input.json").unwrap();
+    let prover_input: ProverInput = serde_json::from_slice(&bytes).unwrap();
+
+    c.bench_function("Block 19240650 processing", |b| {
```

Comment:
This string is used to identify the test, notably as a CLI argument. I'd recommend something simpler like `block_processing`.

Reply:
What do you mean as a CLI argument?

Reply:
What do you mean as a CLI argument? The benchmark name is `block_processing`

Reply:
Hmm, I can't run it with `block_processing`, I have to put a substring of this like `cargo bench "Block 19240650"`.

Reply:
Ah, nevermind, I wasn't using `--bench`.